.eslintrc.json

```diff
@@ -6,9 +6,10 @@
     "next/core-web-vitals",
     "prettier"
   ],
-  "rules":{
+  "rules": {
     // @TODO: ideally, these should be removed
     "@typescript-eslint/no-explicit-any": "off",
     "import/named": "off"
-  }
+  },
+  "ignorePatterns": ["./temp/*"]
```

Comment:
Adding these only while we're testing the Stellar XDR package locally in the repo. These ESlint updates will be removed once we use the npm package.